index.js

```diff
@@ -29,14 +27,14 @@ function main(fn, fp, data, opts) {
 		});
 	}
 
-	var json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
+	const json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
 
 	return fn(fp, json, {mode: opts.mode});
 }
 
 module.exports = function (fp, data, opts) {
```

Comment:
@LinusU Yes, but you see in the stack trace which file it's from, so it's clear it's the main export from that file.